src/opendap/auth/UrsIdPTest.java

```diff
@@ -0,0 +1,211 @@
+/*
+* /////////////////////////////////////////////////////////////////////////////
+* This file is part of the "Hyrax Data Server" project.
+*
+*
+* Copyright (c) 2025 OPeNDAP, Inc.
+* Author: Nathan David Potter  <ndp@opendap.org>
+*
+* This library is free software; you can redistribute it and/or
+* modify it under the terms of the GNU Lesser General Public
+* License as published by the Free Software Foundation; either
+* version 2.1 of the License, or (at your option) any later version.
+*
+* This library is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+* Lesser General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public
+* License along with this library; if not, write to the Free Software
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
+*
+* You can contact OPeNDAP, Inc. at PO Box 112, Saunderstown, RI. 02874-0112.
+* /////////////////////////////////////////////////////////////////////////////
+*/
+
+package opendap.auth;
+
+import opendap.auth.UrsIdP;
+
+import com.auth0.jwk.InvalidPublicKeyException;
+import com.google.gson.JsonParseException;
+import java.math.BigInteger;
+import java.security.interfaces.RSAPublicKey;
+import java.util.Base64;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class UrsIdPTest {
+    // Public test token and JWKS provided by EDL:
+    // https://urs.earthdata.nasa.gov/documentation/for_integrators/verify_edl_jwt_token
+    public static final String EDL_TEST_JWKS = "{" +
```

Comment:
Seems like our version of Java complains about multiline strings, or perhaps I was just constructing them incorrectly? I attempted to do `"""{"keys" ...` but no dice.